.github/workflows/tests.yaml

```diff
@@ -14,14 +14,15 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
+          submodules: true
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
           python-version: "3.10"
```

Comment:
There's a way in the python installer action to cache pip downloads - might be useful here.

I *think* it might even be able to cache the built wheel so we're not rebuilding `python-apt` each time.

Reply:
We can apply those changes uniformly to all projects to avoid having an outlier.